src/raftstore/store/peer.rs

```diff
@@ -401,7 +402,11 @@ impl Peer {
         ));
         // write kv rocksdb first in case of restart happen between two write
         try!(self.kv_engine.write(kv_wb));
-        try!(self.raft_engine.write(raft_wb));
+        try!(self.kv_engine.flush_wal(self.cfg.sync_log));
```

Comment:
need sync here even we enable sync_log?

Reply:
yes,  write kv data to engine first, not means it will sync to disk first